thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/AnomalyDetectionInputContext.java

```diff
@@ -49,4 +50,12 @@ public void setKnownMergedAnomalies(ListMultimap<DimensionMap, MergedAnomalyResu
   public void setScalingFactors(List<ScalingFactor> scalingFactors) {
     this.scalingFactors = scalingFactors;
   }
+
+  public MetricTimeSeries getMetricTraffic() {
```

Comment:
What's the relation of this metric to other TimeSeries? Can we find a better way to pass these information when do dimension drill down? for example use metricName --> Timeseries mapping? Should we name it differently?